tests/ttnn/unit_tests/operations/test_maxpool2d.py

```diff
@@ -805,3 +834,43 @@ def test_pool_core_nondivis(
     assert isclose
     if dtype == ttnn.bfloat16:
         assert isequal
+
+
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 24576}], indirect=True)
+@pytest.mark.parametrize(
+    "act_shape",
+    (([1, 256, 54, 54],)),
+)
+@pytest.mark.parametrize(
+    "kernel_size",
+    ((3, 3),),
+)
+@pytest.mark.parametrize(
+    "padding",
+    ((0, 0),),
+)
+@pytest.mark.parametrize("stride", ((2, 2),))
+@pytest.mark.parametrize("dilation", ((1, 1),))
+@pytest.mark.parametrize("dtype", [ttnn.bfloat16])
+@pytest.mark.parametrize("ceil_mode", [True])
```

Comment:
can we test with both True and False?

Reply:
The SqueezeNet model specifically uses ceil_mode=True.  There is significant test coverage elsewhere for ceil_mode=True/False, but I would be happy to add the extra parameterization here as well if you would prefer.  Let me know what you think is best!